packages/core/src/model.js

```diff
@@ -119,7 +170,7 @@ export class Model extends ModelTypeScript {
     this._previousDataValues = {};
     this.uniqno = 1;
     this._changed = new Set();
-    this._options = omit(options, ['comesFromDatabase']);
```

Comment:
I think that this (using `omit` instead of `_.omit`) is actually the preferred solution, it's also how we've configured eslint in combination with eslint-plugin-small-import on .ts files. I think if we merge a version of this PR, we should use this approach for .js files as well and enable that in the eslint config